### ISSUE TITLE
Request: Reorder pages

### ISSUE THREAD

**Issue by S-Joao-V** [Akylas/com.akylas.documentscanner#46]
During a scan sometimes you capture something in the wrong order, reordering would be a useful feature.
Long press + drag should be almost intuitive when browsing the document pictures.

**Comment by farfromrefug** [comment]
@S-Joao-V Not good if the order is not respected. And seems weird. You are talking about using the batch button from the camera view to scan multiple? Then the order is not the order you took picture?

About reordering it is already possible. You can long press the page number. It is not perfect but i cant use longpress+drag as longpress is already used for selecting items.
Will try to think of a better way

**Comment by S-Joao-V** [comment]
OK! Dragging by the page number was what I meant.
Its difficult to get to the drag mode due to its size, the whole picture frame or a little handle could be used for something more intuitive.

Dec 28, 2023 11:02:36 farfromrefuge ***@***.***>:

@S-Joao-V<https://github.com/S-Joao-V> Not good if the order is not respected. And seems weird. You are talking about using the batch button from the camera view to scan multiple? Then the order is not the order you took picture?

About reordering it is already possible. You can long press the page number. It is not perfect but i cant use longpress+drag as longpress is already used for selecting items.
Will try to think of a better way

**Comment by farfromrefug** [comment]
@S-Joao-V indeed using the image could work but then it would be hard to select the item as you would have to long press outside the image. not sure what to do here

**Comment by farfromrefug** [comment]
@S-Joao-V there is a new way to reorder pages now. Simply start dragging the item horizontally

**Comment by S-Joao-V** [comment]
@farfromrefug tks! Will check it out

**Comment by revsuine** [comment]
Perhaps a "reorder mode" that can be entered via the three dots menu in the top right would work in terms of "hitbox size"?

**Comment by farfromrefug** [comment]
@revsuine yes that would be a good first approach. Will try to look at this though I am on a 6 months bycycle trip right now so can't do.much

**Comment by drp4positive** [comment]
Hey Thanks! Just started with OSSDS and it looks nice. Like ScanBot that I used on iOS for a decade or so. But I can not figure out the "reorder page" feature... tried everything I can think of. I am on Pixel 8 with the latest GrapheneOS. Thanks for your help.

**Comment by farfromrefug** [comment]
@drp4positive sorry it is really hidden right now. You need to longpress the page number.
Need to fix that !

**Comment by farfromrefug** [comment]
@drp4positive implemented

**Comment by drp4positive** [comment]
Thanks. Works fine now 👍️
